src/dftbp/extlibs/poisson.F90

```diff
@@ -35,6 +34,9 @@ module dftbp_extlibs_poisson
       & set_poissonbox, set_poissongrid, set_accuracy, set_verbose, check_biasdir,&
       & check_poisson_box, check_parameters, check_localbc, check_contacts, write_parameters,&
       & poiss_getlatvecs
+#:if WITH_MPI
```

Comment:
Not sure, but indenting the inner macro might be easier to read (but leave the actual use statement in place).

Reply:
Yeah, I wasn't sure either. I've indented the macro by two spaces now.